2007-05-02  Jakub Jelinek  <jakub@redhat.com>

	PR libgomp/28482
	* configure.tgt: Don't link with -Wl,-z,nodlopen even on Linux.

--- libgomp/configure.tgt.jj	2007-04-20 12:55:40.000000000 +0200
+++ libgomp/configure.tgt	2007-05-02 17:32:15.000000000 +0200
@@ -11,11 +11,8 @@
 #  XLDFLAGS		Add extra link flags to use.
 
 # Optimize TLS usage by avoiding the overhead of dynamic allocation.
-# This does require that the library be present during process 
-# startup, so mark the library as not to be dlopened.
 if test $have_tls = yes && test "$with_gnu_ld" = "yes"; then
 	XCFLAGS="${XCFLAGS} -ftls-model=initial-exec"
-	XLDFLAGS="${XLDFLAGS} -Wl,-z,nodlopen"
 fi
 
 # Since we require POSIX threads, assume a POSIX system by default.
